articles/zk-snarks-under-the-hood/zk_snarks_under_the_hood.md

```diff
@@ -0,0 +1,139 @@
+---
+Title: "Zk-SNARKs: Under the Hood"
+Status: Done
+Level: 6
+Note: There is a few places that are hard to fully understand
+---
+
+# Introduction
+
+Remember that the solution to a QAP is a set of polynomials (A, B, C) such that A(x) * B(x) - C(x) = H(x) * Z(x), where:
+
+- A is a linear combination of a set of polynomials {A_1…A_m}
+- B is the linear combination of {B_1…B_m} with the same coefficients
```

Comment:
Same coefficients

Reply:
what do you want me to change here? Is it "the same" to "same" ?

Reply:
I mean the same coefficients with what?